ruby_on_rails/rspec.md

```diff
@@ -12,9 +12,11 @@ group :development, :test do
 end
 
 group :test do
+  gem 'capybara'
   gem 'factory_bot_rails'
   gem 'faker'
   gem 'shoulda-matchers'
+  gem 'selenium-webdriver'
```

Comment:
they are already provided by rails. does maybe `--skip-tests` influence this as well?

Reply:
Ah, this can be.

Reply:
Yes, it's the case. I'll undo the skip-test